projects/bp-gallery/src/components/views/picture/sidebar/picture-info/PictureEditDialog.tsx

```diff
@@ -16,15 +16,15 @@ const isDefaultCropZone = (rect: any) => {
   return rect.width < 1 || rect.height < 1;
 };
 
-const PictureEditDialog = ({
+const PictureEditDialog = memo(function PictureEditDialog({
```

Comment:
Is there a reason you chose to convert the arrow function to a plain function?

Reply:
getting a "Component definition is missing display name" warning if I try to use an arrow function there and decided that just using a function is probably better than adding a new eslint rule or ignore comment

Reply:
Ah, thanks!